index.html

```diff
@@ -0,0 +1,154 @@
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, intial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+		<link href="style.css" rel="stylesheet" type="text/CSS">
+		<title >Robin Hodge's Portfolio Website</title>
+	</head>
+	<body id="top">
+		<div class="contact">
+			<div class="contact-outer-container contact-top">
+				<h2>Contact</h2>
+				<p> The best way to get in touch with me is via email, but if you prefer social media I have included these too.</p>
+				<div class="contact-container">
+					<a href="mailto:mrrobinhodge@gmail.com"><img src="email.png" alt="An email thumbnail"></a>
+					<a href="https://www.linkedin.com/in/robin-hodge/" target="_blank"><img src="linkedin.png" alt="A linkedin thumbnail"></a>
+					<a href="https://github.com/horsenelly" target="_blank"><img src="github.png" alt="A github thumbnail"></a>
+				</div>
+			</div>
+		</div>
+		</div>
+		<nav id="nav">
+			<div class="nav-left-container">
+				<a href="#about">About</a>
+				<a href="#portfolio">Portfolio</a>
+				<a href="#contact">Contact</a>
+				<a href="https://docs.google.com/document/d/1G8ed6eSbtD6e5KjoC9QAf5T7TktOjEkubLuD5ih2ifs/edit" target="_blank">My CV</a>
+		</nav>
+		<div class="intro">
+			<img src="https://images.mubicdn.net/images/cast_member/2552/cache-207-1524922850/image-w856.jpg" alt="An image of me"> 
+			<div class="intro-texts">
+				<h2>Hi!<h2>
+				<h1>Welcome to <br>Robin's portfolio</h1>
+				<h3>I am an aspiring full stack developer, and recent graduate from iO Academy</h3>
+			</div>
+		</div>
+		<div class="about" id="about">
+			<div class="about-me">
```

Comment:
here again you have multiple layers where you can achieve the same layout by having only the outer one and putting all their css properties together. so basically, you keep the outer div and you take all the styling from the inner div and put it on the outer div

Reply:
resolved!